themes/digital.gov/layouts/partials/core/get_authors.html

```diff
@@ -35,7 +35,8 @@
           <div class="grid-col-9">
             <div class="details">
               <h5>
-                <a href="{{- (printf "authors/%s/" $uid) | absURL -}} " title="Posts by {{- $author.Params.display_name | default $uid -}} " rel="author">
+                <span class="usa-sr-only">Orginally posted by {{ $author.Params.display_name | default $uid }} on {{ $.Date.Format "Jan 2, 2006" }}</span>
```

Comment:
Typo in `Originally`.

```suggestion
                <span class="usa-sr-only">Originally posted by {{ $author.Params.display_name | default $uid }} on {{ $.Date.Format "Jan 2, 2006" }}</span>
```

Reply:
Fixed